opentimestamps/core/secp256k1.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (C) 2017 The OpenTimestamps developers
+#
+# This file is part of python-opentimestamps.
+#
+# It is subject to the license terms in the LICENSE file found in the top-level
+# directory of this distribution.
+#
+# No part of python-opentimestamps including this file, may be copied,
+# modified, propagated, or distributed except according to the terms contained
+# in the LICENSE file.
+
+import hashlib
+
+from opentimestamps.core.op import UnaryOp, MsgValueError
+
+@UnaryOp._register_op
+class OpSecp256k1Commitment(UnaryOp):
+    """Map (P || commit) -> [P + sha256(P||commit)G]_x for a given secp256k1 point P
```

Comment:
This description isn't actually correct: the opcode isn't taking a "commit", but rather an arbitrary message, limited only by the `MAX_MSG_LENGTH` limitation.

So I'd suggest we change that line to:

    Map (P || m) -> [P + sha256(P || m)G]_x for a given secp256k1 point P

Reply:
Sure, I'll reword this. I had meant "commit" in the sense of "thing that is being committed to".